library/solr

```diff
@@ -1,125 +1,140 @@
 Directory: 7.3
 
 Tags: 7.3.1-alpine, 7.3-alpine
 Architectures: amd64, arm32v6, arm64v8, i386, ppc64le, s390x
-GitCommit: 9319e4b65c33c47dd5f901daec9d457571294396
+GitCommit: d61313b443fb798a3a4e873d2255216e7b5c3a69
 Directory: 7.3/alpine
 
 Tags: 7.3.1-slim, 7.3-slim
 Architectures: amd64, arm32v5, arm32v7, arm64v8, i386, ppc64le, s390x
-GitCommit: ef954ee10a6483607daa32fc3c35a6ca462c0b3f
+GitCommit: d61313b443fb798a3a4e873d2255216e7b5c3a69
 Directory: 7.3/slim
 
 Tags: 7.2.1, 7.2
 Architectures: amd64, arm32v5, arm32v7, arm64v8, i386, ppc64le, s390x
-GitCommit: 9319e4b65c33c47dd5f901daec9d457571294396
+GitCommit: d61313b443fb798a3a4e873d2255216e7b5c3a69
 Directory: 7.2
 
 Tags: 7.2.1-alpine, 7.2-alpine
 Architectures: amd64, arm32v6, arm64v8, i386, ppc64le, s390x
-GitCommit: 9319e4b65c33c47dd5f901daec9d457571294396
+GitCommit: d61313b443fb798a3a4e873d2255216e7b5c3a69
 Directory: 7.2/alpine
 
 Tags: 7.2.1-slim, 7.2-slim
 Architectures: amd64, arm32v5, arm32v7, arm64v8, i386, ppc64le, s390x
-GitCommit: 9319e4b65c33c47dd5f901daec9d457571294396
+GitCommit: d61313b443fb798a3a4e873d2255216e7b5c3a69
 Directory: 7.2/slim
 
 Tags: 7.1.0, 7.1
 Architectures: amd64, arm32v5, arm32v7, arm64v8, i386, ppc64le, s390x
-GitCommit: 9319e4b65c33c47dd5f901daec9d457571294396
+GitCommit: d61313b443fb798a3a4e873d2255216e7b5c3a69
 Directory: 7.1
 
 Tags: 7.1.0-alpine, 7.1-alpine
 Architectures: amd64, arm32v6, arm64v8, i386, ppc64le, s390x
-GitCommit: 9319e4b65c33c47dd5f901daec9d457571294396
+GitCommit: d61313b443fb798a3a4e873d2255216e7b5c3a69
 Directory: 7.1/alpine
 
 Tags: 7.1.0-slim, 7.1-slim
 Architectures: amd64, arm32v5, arm32v7, arm64v8, i386, ppc64le, s390x
-GitCommit: 9319e4b65c33c47dd5f901daec9d457571294396
+GitCommit: d61313b443fb798a3a4e873d2255216e7b5c3a69
 Directory: 7.1/slim
 
 Tags: 6.6.5, 6.6, 6
 Architectures: amd64, arm32v5, arm32v7, arm64v8, i386, ppc64le, s390x
-GitCommit: 9319e4b65c33c47dd5f901daec9d457571294396
+GitCommit: d61313b443fb798a3a4e873d2255216e7b5c3a69
 Directory: 6.6
 
 Tags: 6.6.5-alpine, 6.6-alpine, 6-alpine
 Architectures: amd64, arm32v6, arm64v8, i386, ppc64le, s390x
-GitCommit: 9319e4b65c33c47dd5f901daec9d457571294396
+GitCommit: d61313b443fb798a3a4e873d2255216e7b5c3a69
 Directory: 6.6/alpine
 
 Tags: 6.6.5-slim, 6.6-slim, 6-slim
 Architectures: amd64, arm32v5, arm32v7, arm64v8, i386, ppc64le, s390x
-GitCommit: 9319e4b65c33c47dd5f901daec9d457571294396
+GitCommit: d61313b443fb798a3a4e873d2255216e7b5c3a69
 Directory: 6.6/slim
 
 Tags: 5.5.5, 5.5, 5
 Architectures: amd64, arm32v5, arm32v7, arm64v8, i386, ppc64le, s390x
-GitCommit: 9319e4b65c33c47dd5f901daec9d457571294396
+GitCommit: d61313b443fb798a3a4e873d2255216e7b5c3a69
 Directory: 5.5
 
 Tags: 5.5.5-alpine, 5.5-alpine, 5-alpine
 Architectures: amd64, arm32v6, arm64v8, i386, ppc64le, s390x
-GitCommit: 9319e4b65c33c47dd5f901daec9d457571294396
+GitCommit: d61313b443fb798a3a4e873d2255216e7b5c3a69
 Directory: 5.5/alpine
 
 Tags: 5.5.5-slim, 5.5-slim, 5-slim
```

Comment:
Any particular reason to keep so many versions listed here?  Removing them here will remove them from the "Supported" section on the Hub readme (and will prevent us from spending cycles rebuilding them on the official build server), but the tags will still be available to users who want them.  (See https://github.com/docker-library/official-images#library-definition-files for more detail on this.)